pom.xml

```diff
@@ -274,7 +274,7 @@
         <gravitee-entrypoint-http-get.version>1.2.0</gravitee-entrypoint-http-get.version>
         <gravitee-entrypoint-http-post.version>1.2.0</gravitee-entrypoint-http-post.version>
         <gravitee-entrypoint-sse.version>4.1.0</gravitee-entrypoint-sse.version>
-        <gravitee-entrypoint-webhook.version>3.0.0</gravitee-entrypoint-webhook.version>
+        <gravitee-entrypoint-webhook.version>3.0.1-apim-6600-handle-malformed-messages-SNAPSHOT</gravitee-entrypoint-webhook.version>
```

Comment:
Don't forget to replace with the released version

Reply:
Done